src/views/base.html

```diff
@@ -25,11 +25,11 @@
       if the user variable is available (only the case if the page is authenticated) let them know they're logged in via a header
     #}
     {% if user.firstName %}
-      <header class="logged-in">
-        Logged in as {{ user.firstName }} {{ user.lastName }} &nbsp; <a href="/apply/logout">Logout</a>
+      <header class="logged-in clearfix">
+        <span class="long-text">Logged in as</span> {{ user.firstName }} {{ user.lastName }} &nbsp; <a href="/apply/logout">Logout</a>
         {# dashboard view explicitly sets is_dashboard to true, otherwise is falsey #}
         {% if not is_dashboard %}
-          <a class="float-right" href="/apply/dashboard">Go to your dashboard &rarr;</a>
+          <a class="float-right" href="/apply/dashboard"><span class="long-text">Go to your dashboard</span><span class="short-text">Dashboard</span> &rarr;</a>
```

Comment:
I think the design choice is good, it's just the implementation is making me a bit uncomfortable. CSS hasn't really figured out how to solve these kinds of problems elegantly yet.